src/formio/components/FileField.js

```diff
@@ -291,6 +291,13 @@ class FileField extends Formio.Components.components.file {
     return super.validatePattern(file, val);
   }
 
+  setErrorClasses(elements, dirty, hasErrors, hasMessages) {
+    const input = this.element.querySelector('[ref="fileBrowse"]');
```

Comment:
this should also use `this.refs.fileBrowse`

Reply:
ah okay, yeah if we can reach it via `this` then that would be better. I'll change it 👍